EpiAware/src/EpiLatentModels/Intercept.jl

```diff
@@ -0,0 +1,78 @@
+@doc raw"
+The `Intercept` struct is used to model the intercept of a latent process. It
+broadcasts a single intercept value to a length `n` latent process.
+
+## Constructors
+
+- Intercept(intercept_prior)
+- Intercept(; intercept_prior)
+
+## Examples
+
+```julia
+using Distributions, Turing, EpiAware
+int = Intercept(Normal(0, 1))
+int_model = generate_latent(int, 10)
+rand(int_model)
+int_model()
+```
+"
+@kwdef struct Intercept{D <: Sampleable} <: AbstractTuringLatentModel
```

Comment:
We might want to reason on models that have an intercept, so is it worth defining an abstract type something like

```julia
AbstractTuringIntercept <: AbstractTuringLatentModel
```

then making both the intercepts here a sub-type of `AbstractTuringIntercept`?

Reply:
ah yeah that is a good idea

Reply:
Done [c7e42fb](https://github.com/CDCgov/Rt-without-renewal/pull/203/commits/c7e42fb0f98a5f4c657827617e35aaf1f799df31)